test/runtests.jl

```diff
@@ -1,15 +1,10 @@
 reload("CSV")
 using Base.Test, NullableArrays, DataStreams
 
-### File.jl
-
-# dir = "/Users/jacobquinn/.julia/v0.4/CSV/test/test_files/"
 dir = joinpath(dirname(@__FILE__),"test_files/")
 
-# test 0.3 and 0.4
-
 #test on non-existent file
-@test_throws ArgumentError CSV.Source("");
+@test_throws ArgumentError CSV.Source(Compat.UTF8String(""));
```

Comment:
While you're at it, it would be cool to make `CSV.Source` accept any `AsbtractString`, and convert it to `UTF8String` automatically.